src/configs/OnboardingList.config.js

```diff
@@ -38,13 +40,36 @@ const signUpBluehostAcademy = () => {
   AnalyticsSdk.track("next_step", "next_step_bh_wp_academy_clicked", data);
   WordPressSdk.settings.put({ bluehost_academy_signup_clicked: true });
 };
+
+const updateSiteServers = (setWebServersUpdated) => {
+  AnalyticsSdk.track("next_step", "next_step_update_nameserver_clicked", data);
+  WordPressSdk.settings.put({ update_site_server_clicked: true }).then(() => {
+    WordPressSdk.settings.get();
+    setWebServersUpdated(true);
+  });
+};
+const updateSiteDomain = () => {
+  AnalyticsSdk.track("next_step", "next_step_connect_domain_clicked", data);
+};
+const updateMigrateViewGuide = () => {
+  AnalyticsSdk.track("next_step", "next_step_migrate_view_guide_clicked", data);
+};
+
+const updateStoreSetup = (setIsMigrationCompleted) => {
+  WordPressSdk.settings.put({ showMigrationSteps: false }).then(() => {
+    WordPressSdk.settings.get();
+    setIsMigrationCompleted(false);
+  });
+};
 const signUpYoastSEOAcademy = () => {
   WordPressSdk.settings.put({ yoast_seo_signup_status: true });
   AnalyticsSdk.track("next_step", "next_step_yoast_academy_clicked", data);
 };
 const brandName =
```

Comment:
check if `brandName` variable could also be added to constants, and imported at necessary places